src/selectors/data/__specs__/bookingsSelectors.spec.js

```diff
@@ -332,12 +332,48 @@ describe('selectFinishedEventBookings', () => {
       // when
       const results = selectFinishedEventBookings(state)
 
+      // then
+      expect(results).toHaveLength(0)
+    })
+
+    it('should return bookings that are not bookable and expired', () => {
```

Comment:
```suggestion
    it('should return bookings that are not bookable but expired', () => {
```
J'ai toujours du mal à saisir le sens dans ce cas précis, on pourrait croire que la négation s'effectue aussi sur `expired`.
Qu'en penses tu ?

Reply:
:+1: